.circleci/config.yml

```diff
@@ -89,20 +97,52 @@ jobs:
       - restore_cache: *restore_cache
       - run: *install_bundler
       - save_cache: *save_cache
-      - run: *run_lint
       - run: *run_tests
       - store_test_results: *store_test_results
       - store_artifacts: *store_artifacts
 
+  release:
+    description: Release a new version of the Ruby client
+    parameters:
+      version:
+        type: string
+    docker:
+      - *default_docker_ruby_executor
+    steps:
+      - checkout
+      - run: *check_bundler
+      - run: *install_bundler
+      - run:
+          name: Build new gem
+          command: gem build algolia.gemspec
+      - run:
+          name: Publish new gem
+          command: |
+            if [[ -z "$RUBYGEMS_API_KEY" ]]; then echo '$RUBYGEMS_API_KEY is not set'; exit 1; fi
+            gem push --key $RUBYGEMS_API_KEY $(ls algolia-*.gem)
+
 workflows:
   version: 2
   ci:
     jobs:
+      - format:
+          version: '2.7.1'
       - test_ruby:
           matrix:
             parameters:
-              version: ['2.2', '2.3', '2.4', '2.5', '2.6', 'latest']
+              version: ['2.2.10', '2.3.8', '2.4.10', '2.5.8', '2.6.6', '2.7.1']
```

Comment:
Why do we have the version fixed to e.g. `2.5.8` instead of `2.5`? If I remember correctly, using `2.5` will always take the latest version of `2.5.x`, so if there's (for example) a security flaw that is fixed and `2.5.9` is released, we'll test against that version instead of `2.5.8`.

Reply:
Since I wasn't sure the latest patch version was used, I've preferred to set them explicitly. This indeed has the drawback you mention of not updating to the latest possible version.

I know maintainers have different views regarding the explicit vs. implicit declarations so I'll let you @DevinCodes and @chloelbn tell me what you prefer here, since you'll be the maintainers, and I'll either keep or remove the commit accordingly.

Reply:
My vote goes to implicit declarations here.

What about you @chloelbn ?

Reply:
Now that the problem with the older versions is fixed I'd rather go back to implicit as well

Reply:
Just changed it. Thanks for your reviews 👍 

I've kept `2.7` instead of `latest` to prevent us from having to look on Docker Hub to understand which version is used. Same for JRuby.